fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/impl/LuceneSearchProvider.java

```diff
@@ -587,6 +591,7 @@ private Query getFilterQuery(Set<ApiFilter> filters, int perPage) {
             throw new RuntimeException(e);
```

Comment:
Can't comment in the right place, but there are effectively 2 steps in this search operation:

1. Get the Lucene results for the right page
2. Hydrate the dim rows

I would wrap a timer around each of those high-level steps